org.kde.kget.json

```
@@ -0,0 +1,140 @@
+          "sha256" : "eebc3c1b27f1c8979896ff361ba9bb4778b508b2496c2fc10e3775a40b1de1ad",
+          "type" : "archive"
+        }
+      ],
+      "cleanup" : [
+        "/lib/libgpgme*.la",
+        "/lib/libqgpgme.la"
+      ],
+      "modules" : [
+        {
+          "name" : "libgpg-error",
+          "sources" : [
+            {
+              "url" : "https://gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.42.tar.bz2",
+              "sha256" : "fc07e70f6c615f8c4f590a8e37a9b8dd2e2ca1e9408f8e60459c67452b925e23",
+              "type" : "archive"
+            }
+          ],
+          "cleanup" : [
+            "/lib/libgpg-error.la"
+          ],
+          "modules" : [
+            {
+              "name" : "gettext",
+              "sources" : [
+                {
+                  "url" : "https://ftp.gnu.org/pub/gnu/gettext/gettext-0.19.8.tar.gz",
+                  "sha256" : "3da4f6bd79685648ecf46dab51d66fcdddc156f41ed07e580a696a38ac61d48f",
+                  "type" : "archive"
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "name" : "libassuan",
+          "sources" : [
+            {
+              "url" : "https://gnupg.org/ftp/gcrypt/libassuan/libassuan-2.5.5.tar.bz2",
+              "sha256" : "8e8c2fcc982f9ca67dcbb1d95e2dc746b1739a4668bc20b3a3c5be632edb34e4",
+              "type" : "archive"
+            }
+          ],
+          "cleanup" : [
+            "/lib/libassuan.la"
+          ]
+        }
+      ]
+    },
+    {
+      "name" : "kdelibs4support",
+      "buildsystem" : "cmake-ninja",
+      "config-opts" : [
+        "-DENABLE_TESTING=OFF"
+      ],
+      "sources" : [
+        {
+          "url" : "https://invent.kde.org/frameworks/kdelibs4support.git",
+          "commit" : "65eedb15cc81391f72a7dbebcb6861c46a58fb45",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "qca",
+      "buildsystem" : "cmake-ninja",
+      "sources" : [
+        {
+          "url" : "https://invent.kde.org/kde/qca.git",
+          "commit" : "ce14c3c6971476035b678058f0511b35b671d27e",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "libktorrent",
+      "buildsystem" : "cmake-ninja",
+      "sources" : [
+        {
+          "url" : "https://invent.kde.org/kde/libktorrent.git",
+          "commit" : "d34f71aec7ef346568421c07abe839eba1597035",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "kget",
+      "buildsystem" : "cmake-ninja",
+      "sources" : [
+        {
+          "url" : "https://invent.kde.org/kde/kget.git",
+          "commit" : "a8d2ff6cc51afecdbe962378dafb1c6fe220dece",
+          "type" : "git"
+        }
+      ]
+    }
+  ],
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=host",
+    "--share=network"
+  ],
+  "tags" : [
+    "nightly"
+  ],
+  "rename-icon" : "kget",
+  "desktop-file-name-suffix" : " (Nightly)"
```

all of these are usually toward the top of the manifest.

```suggestion
  "finish-args" : [
    "--share=ipc",
    "--socket=fallback-x11",
    "--socket=wayland",
    "--filesystem=host",
    "--share=network"
  ],
  "rename-icon" : "kget"
```